airbyte-integrations/bases/base-normalization/integration_tests/normalization_test_output/mysql/test_nested_streams/first_output/airbyte_incremental/test_normalization/nested_stream_with_co___names_partition_data.sql

```diff
@@ -20,7 +20,7 @@ with numbers as (
     select
 
     
-    p0.generated_number * pow(2, 0)
+    p0.generated_number * power(2, 0)
```

Comment:
`pow` and `power` are apparently equivalent https://stackoverflow.com/a/51615601